rp2040-hal/examples/watchdog.rs

```diff
@@ -21,8 +21,9 @@ use rp2040_hal as hal;
 use hal::pac;
 
 // Some traits we need
-use embedded_hal::digital::v2::OutputPin;
-use embedded_hal::watchdog::{Watchdog, WatchdogEnable};
+use embedded_hal::digital::OutputPin;
+// Embedded HAL 1.0.0 doesn't have an Watchdog trait, so use the one from 0.2
+use embedded_hal_0_2::watchdog::{Watchdog, WatchdogEnable};
```

Comment:
We should probably ensure that any e-h 0.2 related functionality is available at inherent methods so you don't *have* to use eh 0.2. It's not like forcing people to use e-h 0.2 gives you portability, because I expect most HALs to drop support for it.